.github/workflows/jit.yml

```diff
@@ -93,12 +93,14 @@ jobs:
             compiler: clang
           - target: aarch64-unknown-linux-gnu/gcc
             architecture: aarch64
-            runner: ubuntu-22.04
+            runner: ubuntu-24.04-aarch64
             compiler: gcc
+            if: ${{ github.repository_owner != 'python' }}
```

Comment:
Is this flipped? Don't we only want to run these jobs only want to run these jobs if the org is python?

Reply:
yes, copy/paste and forgot about it :)